core/parsigex/parsigex.go

```diff
@@ -130,3 +145,60 @@ func (m *ParSigEx) Broadcast(ctx context.Context, duty core.Duty, set core.ParSi
 func (m *ParSigEx) Subscribe(fn func(context.Context, core.Duty, core.ParSignedDataSet) error) {
 	m.subs = append(m.subs, fn)
 }
+
+// NewEth2Verifier returns a verifyFunc instance which is responsible to verify the given partial signatures.
+func NewEth2Verifier(eth2Svc eth2client.Service, pubSharesByKey map[core.PubKey][]*bls_sig.PublicKey) func(context.Context, core.PubKey, core.Duty, core.ParSignedData) error {
```

Comment:
Put this `core` package

Reply:
putting this in core package will introduce cyclic dependency

Reply:
core => signing => tblsconv => core